README.md

```diff
@@ -38,6 +47,36 @@ The configuration file is an [HCL](https://github.com/hashicorp/hcl) formatted f
  | `jwt_bundle_file_mode`              | The octal file mode to use when saving a JWT Bundle file.                                                                         | `0600`                                                                                                                                                               |
  | `jwt_svid_file_mode`                | The octal file mode to use when saving a JWT SVID file.                                                                           | `0600`                                                                                                                                                               |
 
+
+`cmd` and `cmd_args` are used in `daemon_mode` to launch the process that will
```

Comment:
Can we convert these to an unordered list?

Reply:
Done